### PR TITLE
Add stellar issue asset article

### PR DESCRIPTION
**Pull request by gmzabos**
- markdown lint ok
- checked for consistency & typos ;-)

### CODE REVIEW

File: _posts/2021-03-09-stellar-issue-asset.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+subtitle: AFFEN goes live
+tags: [poc, blockchain, stellar, token, asset, github]
+author: gmzabos
+---
+
+## Intro - what is Stellar?
+
+[Stellar](http://stellar.org/){:target="_blank"} is an open-source network for currencies and payments. It is public, there is no "real" owner as it is a decentralized network. See the [Intro](https://stellar.org/learn/intro-to-stellar){:target="_blank"} on their website, as it speaks for itself better than i could do.
+
+## Intro - why do we issue our own asset?
```
Comment (cmeissner) [inline _posts/2021-03-09-stellar-issue-asset.md:13]: I would prefer to have a section like `what is an asset` or is this knowledge what can be assumed as present?

File: _posts/2021-03-09-stellar-issue-asset.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+We came up with the idea while experimenting with Stellar (and other blockchains) and decided to create an asset (or "token") to hook it up to [codeaffen](https://github.com/codeaffen){:target="_blank"} on [github](https://github.com/){:target="_blank"}. The token would function within that ecosystem as a reward for the sake of gamification while submitting code & contributing in other ways to the project. Every good role-playing game has its own XP, you get the idea...
```
Comment (cmeissner) [inline _posts/2021-03-09-stellar-issue-asset.md]: I would only place one link into the text like `[codeaffen on github](https://github.com/codeaffen)...` We want visitors to go to our repo overview not to global github.

File: _posts/2021-03-09-stellar-issue-asset.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+The token was named [AFFEN](https://stellar.expert/explorer/public/asset/AFFEN-GDIZF5MAUDQ6TLIK3PNZ2ESOJAZDNZWIB675S4SQ5RWUTZP3B3ONUPMX){:target="_blank"} (german: "monkey") and is represented by that neat and busy code monkey on our [logo](https://codeaffen.org/assets/img/Codeaffen_only_C_wo_bg.png){:target="_blank"}. There is a supply of 1,000,000 code monkeys to give away. AFFEN, anyone?
+
+## Intro - where to start?
+
+Best starting point is the official [documentation](https://developers.stellar.org/docs/issuing-assets/){:target="_blank"}, but there is also great public articles & videos on the topic. I recommend reading at least into the documentation, just to get a feeling what to do and why to do it. From there on...
```
Comment (cmeissner) [inline _posts/2021-03-09-stellar-issue-asset.md]: `[official stellar documentation]` please

Comment (cmeissner) [inline _posts/2021-03-09-stellar-issue-asset.md]: > but there is also great public articles & videos on the topic

but there are also ... regarding this topic

File: _posts/2021-03-09-stellar-issue-asset.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+## Howto - let's go
+
+- You will need an active wallet, which you use to fund two new accounts with. Make sure you have 5 XLM in that wallet which you can transfer out. I used my [Lobstr](https://lobstr.co/){:target="_blank"} wallet for this.
+- Access Stellar [Laboratory](https://laboratory.stellar.org/){:target="_blank"}. Select "public" on the top right corner.
```
Comment (cmeissner) [inline _posts/2021-03-09-stellar-issue-asset.md]: Maybe `[Stellar Laboratory]`

File: _posts/2021-03-09-stellar-issue-asset.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+The token was named [AFFEN](https://stellar.expert/explorer/public/asset/AFFEN-GDIZF5MAUDQ6TLIK3PNZ2ESOJAZDNZWIB675S4SQ5RWUTZP3B3ONUPMX){:target="_blank"} (german: "monkey") and is represented by that neat and busy code monkey on our [logo](https://codeaffen.org/assets/img/Codeaffen_only_C_wo_bg.png){:target="_blank"}. There is a supply of 1,000,000 code monkeys to give away. AFFEN, anyone?
+
+## Intro - where to start?
+
+Best starting point is the official [documentation](https://developers.stellar.org/docs/issuing-assets/){:target="_blank"}, but there is also great public articles & videos on the topic. I recommend reading at least into the documentation, just to get a feeling what to do and why to do it. From there on...
+
+## Howto - let's go
+
+- You will need an active wallet, which you use to fund two new accounts with. Make sure you have 5 XLM in that wallet which you can transfer out. I used my [Lobstr](https://lobstr.co/){:target="_blank"} wallet for this.
```
Comment (cmeissner) [inline _posts/2021-03-09-stellar-issue-asset.md]: Maybe a small description what `XLM` is should be useful

File: _posts/2021-03-09-stellar-issue-asset.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+## Howto - fund ISSUER and DISTRIBUTOR accounts
+
+- Activate the ISSUER account by sending 2 XLM to it through your Stellar wallet. Use the `Public Key` (ISSUER) as recipient.
+- Activate the DISTRIBUTOR account by sending 3 XLM to it through your Stellar wallet. Use the `Public Key` (DISTRIBUTOR) as recipient.
+- Give it a few seconds, then check both accounts on [stellar.expert](https://stellar.expert/explorer/public){:target="_blank"}. Both accounts should be active and present the amount of XLM, which you just transfered. Go on with...
+
+## Howto - establish trustline from DISTRIBUTOR to ISSUER
+
+- Access Stellar [Laboratory](https://laboratory.stellar.org/){:target="_blank"}. Select "public" on the top right corner.
```
Comment (cmeissner) [inline _posts/2021-03-09-stellar-issue-asset.md]: > Stellar [Laboratory]

[Stellar Laboratory]

**Review by cmeissner — CHANGES_REQUESTED** [review]
Please make better links, not only single words in link name.

Some more information for users that are new on the topic will be great.

File: _posts/2021-03-09-stellar-issue-asset.md (hunk 1 above)
Comment (gmzabos) [reply _posts/2021-03-09-stellar-issue-asset.md:13]: I'll skip this one. The intro explains it.

File: _posts/2021-03-09-stellar-issue-asset.md (hunk 2 above)
Comment (gmzabos) [reply _posts/2021-03-09-stellar-issue-asset.md]: OK

File: _posts/2021-03-09-stellar-issue-asset.md (hunk 3 above)
Comment (gmzabos) [reply _posts/2021-03-09-stellar-issue-asset.md]: OK

Comment (gmzabos) [reply _posts/2021-03-09-stellar-issue-asset.md]: OK

File: _posts/2021-03-09-stellar-issue-asset.md (hunk 4 above)
Comment (gmzabos) [reply _posts/2021-03-09-stellar-issue-asset.md]: OK

File: _posts/2021-03-09-stellar-issue-asset.md (hunk 5 above)
Comment (gmzabos) [reply _posts/2021-03-09-stellar-issue-asset.md]: I'll skip this one - documentation provides it (see link in intro)

File: _posts/2021-03-09-stellar-issue-asset.md (hunk 6 above)
Comment (gmzabos) [reply _posts/2021-03-09-stellar-issue-asset.md]: OK - corrected

**Review by cmeissner — APPROVED** [review]

**Review by cmeissner — APPROVED** [review]

**Pull request merged**